src/content/docs/ru/tutorial/2-pages/1.mdx

```diff
@@ -33,12 +34,13 @@ import PreCheck from '~/components/tutorial/PreCheck.astro';
  :::
```

Comment:
It seems that this tip is recognized as a child element of `Steps` and the build step is failing. Try increasing the indentation here.

Reply:
Thank you so much, it worked!